config/examples/stm32c0.config

```diff
@@ -0,0 +1,34 @@
+ARCH?=ARM
+TARGET?=stm32c0
+SIGN?=ED25519
+#Using SHA384 with ED25519 saves about 2kB without SHA256 code
+HASH?=SHA384
+DEBUG?=0
+VTOR?=1
+CORTEX_M0?=1
+NO_ASM?=0
+EXT_FLASH?=0
+SPI_FLASH?=0
+ALLOW_DOWNGRADE?=0
+NVM_FLASH_WRITEONCE?=1
+WOLFBOOT_VERSION?=0
+V?=0
+SPMATH?=1
+RAM_CODE?=1
+DUALBANK_SWAP?=0
+#Max APP size is 4kB
```

Comment:
8kB

Reply:
I think this is correct, as in NVM_FLASH_WRITEONCE the last two sectors are reserved